front/app/containers/Admin/settings/registration/index.tsx

```diff
@@ -62,8 +64,16 @@ const SettingsRegistrationTab = (_props: Props) => {
     });
   };
 
-  const handleSubmit = async (event: React.FormEvent<HTMLFormElement>) => {
-    event.preventDefault();
+  const handleConfigSettingsChange = (propertyPath: string) => (value: any) => {
+    const newAttributesDiff = { ...(attributesDiff || {}) };
+    set(newAttributesDiff || {}, `settings.${propertyPath}`, value);
```

Comment:
Very much nitpicking here, but would try to avoid using `set` as it doesn't type check the object properties